bootcamp_module08/core/tests/test_student_09.py

```diff
@@ -26,3 +26,16 @@ def test_count_substring_none():
     expected_count = 0
     observed_count = count_substring(test_string, test_substring)
     assert expected_count == observed_count
+
+
+def test_count_substring_insensitive():
+    test_string = "ACAGATAGAA"
+    test_substring = "aGa"
+
+    expected_count = 2
+    observed_count = count_substring(test_string, test_substring)
+    assert expected_count == observed_count
+
+
```

Comment:
I think the extra blank lines here are going to be a style violation. I will try to kick the Travis to verify.